lib/releases.js

```diff
@@ -41,7 +41,24 @@ module.exports.findReleases = async ({ app, context }) => {
   return { draftRelease, lastRelease }
 }
 
-module.exports.generateReleaseBody = ({ config, lastRelease, mergedPullRequests }) => {
+const contributorsSentence = ({ commits, pullRequests }) => {
+  const logins = new Set()
+
+  commits.forEach((commit) => {
+    logins.add(commit.committer.login)
+    logins.add(commit.author.login)
```

Comment:
`login` could be undefined if the email address can't be matched to a valid GitHub user.

Reply:
Ah, good catch 👌🏼